contracts/src/v0.8/operatorforwarder/test/Factory.t.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import "./testhelpers/Deployer.t.sol";
+
+contract FactoryTest is Deployer {
+  function setUp() public {
+    _setUp();
+
+    vm.startPrank(s_alice);
+  }
+
+  /**
+   * @dev Test the deployment of a new operator.
+   */
```

Comment:
nit: prefer `///` over `/** **/`

Reply:
I'm just going to remove these comments since the test name says the same thing